src/tests/ftest/datamover/dummy.txt

```diff
@@ -0,0 +1 @@
+dummy file so git reports a diff so tests will actually run
```

Comment:
FWIW, you can avoid this sort of thing by adding `Doc-only: false` to your commit message.

Reply:
I thought Doc-only skipped the test stage. Is this not true?

Reply:
> I thought Doc-only skipped the test stage. Is this not true?

`Doc-only: true` does.  `Doc-only: false` disables it.

Reply:
Oh, I see. Doc-only is auto-detected :)
Thanks!

Reply:
It is, for some _rough_ (surely not complete) value of auto-detected.